tests/Functional/DbalNestedSetTest.php

```diff
@@ -282,6 +282,16 @@ public function testDeleteNode() {
   }
 
   /**
+   * Tests deleting a node with missing values.
+   *
+   * @expectedException \InvalidArgumentException
```

Comment:
looks good, although ->setExpectedException is preferred over @expectedException see https://thephp.cc/news/2016/02/questioning-phpunit-best-practices